DZ_8/main.py

```diff
@@ -0,0 +1,2 @@
+import controller as c
+c.select_action()
```

Comment:
Оставляйте пустую строку.
```suggestion
import controller as c

c.select_action()
```